core/kt-osrd-sncf-signaling/src/main/kotlin/fr/sncf/osrd/signaling/etcs_level2/ETCS_LEVEL2.kt

```diff
@@ -0,0 +1,58 @@
+package fr.sncf.osrd.signaling.etcs_level2
+
+import fr.sncf.osrd.signaling.BlockDiagReporter
+import fr.sncf.osrd.signaling.SigBlock
+import fr.sncf.osrd.signaling.SignalingSystemDriver
+import fr.sncf.osrd.signaling.SignalingTrainState
+import fr.sncf.osrd.sim_infra.api.SigParametersSchema
+import fr.sncf.osrd.sim_infra.api.SigSettingsSchema
+import fr.sncf.osrd.sim_infra.api.SigState
+import fr.sncf.osrd.sim_infra.api.SigStateSchema
+import fr.sncf.osrd.utils.units.Speed
+import fr.sncf.osrd.utils.units.kilometersPerHour
+
+object ETCS_LEVEL2 : SignalingSystemDriver {
```

Comment:
```suggestion

/** PLACEHOLDER CODE copied from TVM signaling */
object ETCS_LEVEL2 : SignalingSystemDriver {
```

It's probably best to mention it somewhere in the code base

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/10147/commits/983dc60fa77a8739dea21dc67bb93a2a03f10265